sdk/src/main/java/com/mongodb/stitch/android/auth/apiKey/ApiKey.java

```diff
@@ -0,0 +1,65 @@
+package com.mongodb.stitch.android.auth.apiKey;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Api Key offers a secondary form of login.
+ */
+public final class ApiKey {
+    private static class Fields {
+        private static final String ID = "_id";
+        private static final String KEY = "key";
+        private static final String NAME = "name";
+        private static final String DISABLED = "disabled";
+    }
+
+    private final String _id;
+    private final String _key;
+    private final String _name;
+    private final boolean _disabled;
+
+    @JsonCreator
+    private ApiKey(@JsonProperty(Fields.ID)
```

Comment:
[nit] This format feels a little funky/hard to read for me

Reply:
I'd consider annotations and parameters on same line, with separate parameters separated by a new line.

Reply:
Yeah or just use the annotations on the fields instead which I think works just the same right?